Lib/gftools/packager.py

```diff
@@ -302,6 +302,12 @@ def package_family(
                 os.remove(fp)
         shutil.copytree(tmp_dir, family_path, dirs_exist_ok=True)
         save_metadata(family_path / "METADATA.pb", metadata)
+        # Format HTML
+        if os.path.exists(os.path.join(family_path, "DESCRIPTION.en_us.html")):
+            with open(os.path.join(family_path, "DESCRIPTION.en_us.html"), "r") as fin:
+                description = format_html(fin.read())
+            with open(os.path.join(family_path, "DESCRIPTION.en_us.html"), "w") as fout:
+                fout.write(description)
```

Comment:
`family_path` is a `pathlib.Path` object so we don't need to use `os.path`. I don't think we need to check if a description exists because `save_metadata` will add one if one does't exist.

```suggestion
        desc_file = family_path / "DESCRIPTION.en_us.html"
        with open(desc_file, encoding="utf-8") as fin:
            description = format_html(fin.read())
        with open(desc_file, "w", encoding="utf-8") as fout:
            fout.write(description)
```